server/channels/app/import_functions.go

```diff
@@ -1274,6 +1334,27 @@ func (a *App) importReplies(rctx request.CTX, data []imports.ReplyImportData, po
 
 	for _, postWithData := range postsWithData {
 		a.updateFileInfoWithPostId(rctx, postWithData.post)
+
+		if postWithData.replyData.FlaggedBy != nil {
+			var preferences model.Preferences
+
+			for _, username := range *postWithData.replyData.FlaggedBy {
+				user := users[strings.ToLower(username)]
```

Comment:
We are guaranteed to find a user right?

Reply:
Yep, those users are included into the argument of `getUsersByUsernames`  in L:1240